types/seamless-immutable/index.d.ts

```diff
@@ -32,8 +32,8 @@ declare namespace SeamlessImmutable {
         prototype?: any;
     }
 
-    interface AsMutableOptions {
-        deep: boolean;
+    interface AsMutableOptions<TDepthBoolean extends boolean = boolean> {
```

Comment:
Just a small semantic comment here, depth sounds like a should be a scalar value indication how deep the mutable should be applied, I would have thought `TDeep` to more accurate

Reply:
Thanks,

I've made the change you suggested.